cronjobs/process_task_mapping_work_items.sh

```diff
@@ -1,4 +1,10 @@
-#!/bin/sh
+#!/usr/bin/env bash
+
+SSH_DIR=$HOME/.ssh
+SRC_PATH=$(dirname "$0")
```

Comment:
Optional comment: I had to look up what `dirname "$0"` does, I wasn't familiar with this command. Any chance we could rename `SRC_PATH` to `CURRENT_DIR` so it is clearer what this line is doing?

Reply:
I don't like `CURRENT_DIR` because to me that implies cwd (current working directory), which would be the directory the script is being executed from not the directory the script lives in. What about `FILE_DIR`?

Reply:
Ah I see. Yes I like `FILE_DIR`!